tests/rptest/services/redpanda_cloud.py

```diff
@@ -681,9 +681,7 @@ def create(self, superuser: Optional[SaslCredentials] = None) -> str:
                 c = self._get_cluster(_cluster_id)
                 self.current.last_status = c['state']
             except Exception as e:
```

Comment:
from discussion with @savex , i'll add a printout of `e`

Reply:
decided to go with exception chaining in git commit 7444a3efea551355ca7f9d8388d3c8ed5b3c7687 so the calling function can handle the exception appropriately instead of having this function have a side-effect of printing an exception to log.